File: pinyougou-sellergoods-interface/src/main/java/cc/pinyougou/sellergoods/service/BrandService.java
package cc.pinyougou.sellergoods.service;

import cc.pinyougou.entity.PageResult;
import cc.pinyougou.pojo.TbBrand;

import java.util.List;

/**
 * Created by myvisi on 2018/4/23.
 */
public interface BrandService {
    public List<TbBrand> findAll();

    public PageResult findPage(int pageNum, int pageSize);

    public int add(TbBrand tbBrand);

    public TbBrand findOne(Long id);

    public void  update(TbBrand tbBrand);

    public  void  delete(Long[] ids);

    public PageResult findPage(TbBrand tbBrand,int pageNum,int pageSize);
}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
/**
 * Created by myvisi on 2018/4/27.
 */

app.service('brandService',function( $http){
    this.search = function(page,rows,searchEntity){
        return    $http.post('../brand/search.do?page='+page+'&rows='+rows,searchEntity);

    }

    this.delete = function(){
        return   $http.get('../brand/delete.do?ids='+$scope.selectIds);
    }

    this.findOne = function(id){
        return $http.get('../brand/findOne.do?id='+id);
    }
    this.add=function(entity){
        return $http.post('../brand/add.do',entity)
    }

    this.update=function(entity){
        return $http.post('../brand/update.do',entity)
    }
    this.findPage = function(page,rows){
        return  $http.get('../brand/findPage.do?page=' + page + '&rows=' + rows);
    }

})
